core-war/src/main/java/org/silverpeas/web/jobdomain/servlets/JobDomainPeasRequestRouter.java

```diff
@@ -599,13 +611,24 @@ public String getDestination(String function, JobDomainPeasSessionController job
           theNewDomain.setDriverClassName("org.silverpeas.core.admin.domain.driver.ldapdriver.LDAPDriver");
           theNewDomain.setPropFileName("org.silverpeas.domains.domain");
           theNewDomain.setAuthenticationServer("autDomain");
+          theNewDomain.setSilverpeasServerURL(URLUtil.getAbsoluteApplicationURL());
           request.setAttribute("domainObject", theNewDomain);
-          request.setAttribute("action", "domainCreate");
+          request.setAttribute("action", DOMAIN_CREATE_FUNCTION);
+          destination = "domainCreate.jsp";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "domainCreate.jsp" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1192)